sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -28,6 +28,7 @@
 #include "vtimer.h"
 
 #include "net/ng_ndp.h"
+#include "net/ng_fib.h"
```

Comment:
I thought about it, but actually that's not necessary. The header is always there and shouldn't do any harm.